### PR TITLE
[media-common] New Chart

### PR DESCRIPTION
**Pull request by DirtyCajunRice**
Initially has radarr/sonarr/lidarr/ombi/organizr/tautulli. Can add bazarr, nzbget, and sabnzbd in the same fashion.
original PR: https://github.com/billimek/billimek-charts/pull/329

### CODE REVIEW

File: .idea/k8s@home charts.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bjw-s) [inline .idea/k8s@home charts.iml:1]: Might need to `.gitignore` this, as not everyone will be using a Jetbrains IDE.

**Comment by bjw-s** [comment]
Looks great! Is there a reason why you put the actual services under the `media-common` folder instead of the top level `charts` folder? I think that could increase findability (is that a word? 🤔)

File: .idea/k8s@home charts.iml (hunk 1 above)
Comment (DirtyCajunRice) [reply .idea/k8s@home charts.iml]: Good catch! Didnt even notice.

**Comment by DirtyCajunRice** [comment]
> Looks great! Is there a reason why you put the actual services under the `media-common` folder instead of the top level `charts` folder? I think that could increase findability (is that a word? )

So i was talking with @onedr0p  and because they are literally just shells with almost no information i had a hard time swallowing how much bulk that puts in the top level for what it actually was. If yall think it should be top level i can move them.

**Comment by DirtyCajunRice** [comment]
After checking, we can move them to the top as replacements for their former versions, but they reference a chart that doesnt exist yet (media-common) and will break the releaser if they are released at the same time as it. 

In addition, the releaser's logic does not allow nesting like that so they have to go to the base level anyway :(.

**Comment by DirtyCajunRice** [comment]
Ok. MVP is ready. Steps would be:

- [ ] merge this to allow the common chart to exist (requirement for all other charts that depend on this)
- [ ] create new MR that replaces each top level chart with the one in this folder, incrementing them to the next major version as to signify a breaking change
- [ ] have someone who uses them add bazarr, nzbget, and sabnzbd

**Review by carpenike — APPROVED** [review]
LGTM

**Review by carpenike — APPROVED** [review]

**Pull request merged**